src/pip/_vendor/vendor.txt

```diff
@@ -1,4 +1,4 @@
-CacheControl==0.12.11  # Make sure to update the license in pyproject.toml for this.
+CacheControl==0.13.0  # Make sure to update the license in pyproject.toml for this.
```

Comment:
`CacheControl` v0.13.1 was released yesterday:
https://github.com/psf/cachecontrol/releases/tag/v0.13.1

In light of that, is it worth updating the PR to use `0.13.1` instead of `0.13.0`?